docs/shared/tutorials/gradle.md

```diff
@@ -0,0 +1,329 @@
+- Configure a task pipeline
+- Use Nx Plugins to automatically configure caching -->
+
+<!-- ## Final Source Code
+
+Here's the source code of the final result for this tutorial.
+
+{% github-repository url="https://github.com/nrwl/gradle-tutorial/tree/final" /%} -->
+
+## Get Started
+
+Make sure that you have [Gradle](https://gradle.org/) installed on your system. Consult [Gradle's installation guide](https://docs.gradle.org/current/userguide/installation.html) for instruction that are specific to your operating system.
+
+To verify that Gradle was installed correctly, run this command:
+
+```shell
+gradle --version
+```
+
+Next, create a `gradle-tutorial` folder to hold the repository.
+
+```shell
+mkdir gradle-tutorial
+cd gradle-tutorial
+```
+
+Then, run the `gradle init` command to generate a default repository:
+
+```{% command="gradle init --type java-application --dsl kotlin" %}
+Enter target Java version (min: 7, default: 21): 21
+
+Project name (default: gradle-tutorial): gradle-tutorial
+
+Select application structure:
+  1: Single application project
+  2: Application and library project
+Enter selection (default: Single application project) [1..2] 2
+
+Generate build using new APIs and behavior (some features may change in the next minor release)? (default: no) [yes, no] no
+
+
+> Task :init
+To learn more about Gradle by exploring our Samples at https://docs.gradle.org/8.7/samples/sample_building_java_applications_multi_project.html
+
+BUILD SUCCESSFUL in 23s
+1 actionable task: 1 executed
+```
+
+Finally, set up git and create your first commit:
+
+```shell
+git init
+git commit -am "initial commit"
+```
+
+You can run the `gradle projects` command to get a list of projects in the repo.
+
+```text {% command="gradle projects" %}
+> Task :projects
+
+------------------------------------------------------------
+Root project 'gradle-tutorial'
+------------------------------------------------------------
+
+Root project 'gradle-tutorial'
++--- Project ':app'
++--- Project ':list'
+\--- Project ':utilities'
+
+To see a list of the tasks of a project, run gradle <project-path>:tasks
+For example, try running gradle :app:tasks
+
+BUILD SUCCESSFUL in 15s
+11 actionable tasks: 11 executed
+```
+
+The repository has two Java libraries (under `list` and `utilities`) that are used in an application located in the `app` folder.
+
+You can build the `app` project by running `gradle app:build`:
+
+```text {% command="gradle app:build" %}
+BUILD SUCCESSFUL in 3s
+21 actionable tasks: 12 executed, 9 up-to-date
+```
+
+## Add Nx
+
+With the `@nx/gradle` plugin, Nx can be easily added to a Gradle repository and Nx will work seamlessly along side Gradle.
+
+Nx offers many features, but at its core, it is a task runner. Out of the box, it can cache your tasks and ensure those tasks are run in the correct order. After the initial set up, you can incrementally add on other features that would be helpful in your organization.
```

Comment:
```suggestion
Nx offers many features, but at its core, it is a task runner. Out of the box, it can cache your tasks and ensure those tasks are run in the correct order. After the initial setup, you can incrementally add other features that would be helpful in your organization.
```